requirements.txt

```diff
@@ -62,7 +62,8 @@ myst-parser==0.18.0
 networkx==2.8
 numpy==1.22.3
 nxontology==0.4.1
-oaklib==0.1.34
+oaklib==0.1.36
+ontoportal-client==0.0.3
```

Comment:
why is this needed?

Reply:
@matentzn I have no idea. The requirements that I'm actually adding are in `requirements-unlocked.txt`. I just `pip -r install requirements-unlocked.txt; pip freeze > requirements.txt`. I imagine that `ontoportal-client` is a transitive dependency of a this newest updated version of OAK.

Reply:
ok its probably an oak depedndency